tests/js/fileMock.js

```diff
@@ -0,0 +1 @@
+export default 'test-file-stub';
```

Comment:
@caseymorrisus could you please add the license comment to this file?

Reply:
@eugene-manuilov Just pushed up these changes, should be good for another review.

Reply:
@caseymorrisus I don't see it has been pushed. Can you check it again? Also, e2e tests failed, could you please take a look at it?